docs/design/design_v1.9.md

```diff
@@ -69,196 +67,246 @@ This means we aim to standardize:
+     - Include a CN, but that can be anything. `kubeadm` uses the `kubernetes-admin` CN.
+ - A kubeconfig file for kubelet to use, `/etc/kubernetes/kubelet.conf`:
+   - Inside this file, a client certificate is generated from the `ca.crt` and `ca.key`. The client cert should:
+     - Be a client certificate (`x509.ExtKeyUsageClientAuth`).
+     - Be in the `system:nodes` organization.
+     - Have the CN `system:node:<hostname-lowercased>`.
+ - A kubeconfig file for controller-manager, `/etc/kubernetes/controller-manager.conf`:
+   - Inside this file, a client certificate is generated from the `ca.crt` and `ca.key`. The client cert should:
+     - Be a client certificate (`x509.ExtKeyUsageClientAuth`).
+     - Have the CN `system:kube-controller-manager`.
+ - A kubeconfig file for scheduler, `/etc/kubernetes/scheduler.conf`:
+   - Inside this file, a client certificate is generated from the `ca.crt` and `ca.key`. The client cert should:
+     - Be a client certificate (`x509.ExtKeyUsageClientAuth`).
+     - Have the CN `system:kube-scheduler`.
 
 Please note that:
 
-1. `ca.crt` is embedded in all the KubeConfig files.
+1. `ca.crt` is embedded in all the kubeconfig files.
 2. If a given kubeconfig file exists, and its content is evaluated compliant with the above specs, the existing file will be used and the generation phase for the given kubeconfig skipped.
-3. If `kubeadm` is running in "ExternalCA" mode, all the required kubeconfig must be provided by the user as well, because  `kubeadm` cannot generate any of them by itself.
+3. If `kubeadm` is running in ExternalCA mode, all the required kubeconfig must be provided by the user as well, because  `kubeadm` cannot generate any of them by itself.
 4. In case of `kubeadm`  executed in the `--dry-run` mode, kubeconfig files are written in a temporary folder.
-5. KubeConfig files generation can be invoked individually with the `kubeadm phase kubeconfig all` command.
+5. Kubeconfig files generation can be invoked individually with the `kubeadm alpha phase kubeconfig all` command.
 
-### Generate Static Pod Manifest for control plane
+### Generate static Pod manifests for control plane
 
 Common properties for the control plane components:
 
-- `hostNetwork: true` is present on all static pods since there is no network configured yet; accordingly 
-  -  the `address` of the api server for controller-manager and the scheduler will be set to `127.0.0.1`
-  -  if using a local etcd server, `etcd-servers` address  will be set to `127.0.0.1:2379`
-- Leader election is enabled for both the controller-manager and the scheduler
-- controller-manager and the scheduler will reference kubeconfig files with their respective, unique identities
-- Any extra flags specified by the user
+- All static Pods are deployed on `kube-system` namespace.
+- All static Pods gets `tier:control-plane` and `component:{component-name}` labels.
+- All static Pods gets `scheduler.alpha.kubernetes.io/critical-pod` annotation. 
```

Comment:
You can note that this will be moved over to the proper solution of using Pod Priority and Preemption when ready